File: piramid.c
#include <stdio.h>
#include <string.h>

void create_piramid( int rows ) {
	int spaces = rows;
	int i = 0;
	for( i; i < rows * 2; i++ ) {
		int j = 0;
		int n = 0;
		if( ! ( i % 2 ) )
			continue;
		for( j; j < spaces; j++){
			printf(" ");
		}
		for( n; n < i; n++) {
			printf("*");
		}
		spaces--;
		printf("\n");
	}
}

int main() {
	create_piramid( 30 );
	return 0;
}

File: piramid.js
var create_piramid = function( rows ) {
	var spaces = rows;
	for( var i = 0; i < rows * 2; i++ ) {
		if( 0 === ( i % 2) )
			continue;
		var _spaces = '',
			_aterisks = '';
		for( var j = 0; j < spaces; j++)
			_spaces += ' ';
		for( var n = 0; n < i; n++)
			_aterisks += '*';
		console.log( _spaces + _aterisks);
		spaces--;
	}
};
create_piramid( 30 );